plugins/catalog-backend/src/ingestion/processors/CodeOwnersProcessor.ts

```diff
@@ -29,6 +29,8 @@ import { filter, get, head, pipe, reverse } from 'lodash/fp';
 import { Logger } from 'winston';
 import { CatalogProcessor } from './types';
 
+const ALLOWED_KINDS = ['API', 'Component', 'Resource', 'System'];
```

Comment:
I wonder if we should also add Domains?

Reply:
Yeah, indeed.

Reply:
I thought about that, but for our use case we are starting out with domains defined in a single repo location. At least until we decide where the definitions should go. I think for others it's more likely these would exists elsewhere. But again, I guess if `owner` is provided the processor ignores it so I'd also be fine adding it.

Reply:
Yeah exactly - you'd give them manually in that case.